araig_test_gui/CMakeLists.txt

```diff
@@ -0,0 +1,206 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(araig_test_gui)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```

Comment:
Please clean up all comments. Refer from other packagse to see how it should look.